plans/developer-id-distribution.html

```
@@ -0,0 +1,297 @@
+
+<pre><code>xcodebuild archive \
+  -scheme Vellum -configuration Release \
+  -archivePath build/Vellum.xcarchive
+
+xcodebuild -exportArchive \
+  -archivePath build/Vellum.xcarchive \
+  -exportOptionsPlist ExportOptions.plist \
+  -exportPath build/export
+# ExportOptions.plist: method = developer-id
+
+# package (create-dmg or hdiutil) -> build/Vellum.dmg
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)